### ISSUE TITLE
Can't install brew on 10.13.5

### ISSUE THREAD

**Issue by romu70** [Homebrew/brew#4338]
**Please note we will close your issue without comment if you delete, do not read or do not fill out the issue checklist below and provide ALL the requested information. If you repeatedly fail to use the issue template, we will block you from ever submitting issues to Homebrew again.**

- [ ] are reporting a bug others will be able to reproduce and not asking a question. If you're not sure or want to ask a question do so on our Discourse: https://discourse.brew.sh
- [ ] ran a `brew` command and reproduced the problem with multiple formulae? If it's a problem with a single, official formula (not cask) please file this issue at Homebrew/homebrew-core: https://github.com/Homebrew/homebrew-core/issues/new/choose. If it's a `brew cask` problem please file this issue at https://github.com/Homebrew/homebrew-cask/issues/new/choose. If it's a tap (e.g. Homebrew/homebrew-php) problem please file this issue at the tap.
- [ ] ran `brew update` and can still reproduce the problem?
- [ ] ran `brew doctor`, fixed all issues and can still reproduce the problem?
- [ ] ran `brew config` and `brew doctor` and included their output with your issue?

To help us debug your issue please explain:
- What you were trying to do (and why)

I'm just trying to install brew to get wget (and other utilities) on my mac.

- What happened (include command output)

Running ```/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"```leads to:

```
==> This script will install:
/usr/local/bin/brew
/usr/local/share/doc/homebrew
/usr/local/share/man/man1/brew.1
/usr/local/share/zsh/site-functions/_brew
/usr/local/etc/bash_completion.d/brew
/usr/local/Homebrew

Press RETURN to continue or any other key to abort
==> Downloading and installing Homebrew...
error: could not lock config file /usr/local/Homebrew/.git/config: Permission denied
error: could not lock config file /usr/local/Homebrew/.git/config: Permission denied
error: could not lock config file /usr/local/Homebrew/.git/config: Permission denied
error: could not lock config file .git/config: Permission denied
Failed during: git config remote.origin.url https://github.com/Homebrew/brew
TAVEL:~ rtisserand$ sudo chown -R $(whoami) $(brew --prefix)/
Password:
chown: /usr/local/: Operation not permitted
```

- What you expected to happen

brew correctly installed I guess

- Step-by-step reproduction instructions (by running `brew` commands)

Do I really copy/paste the command above? :p

**Comment by romu70** [comment]
Yep, but this may deserve an issue on the brew doc, don't you think?

**Comment by MikeMcQuaid** [comment]
The command that works for you does not work for everyone and the prior is more reliable.
